polaris-react/src/components/SettingToggle/SettingToggle.stories.tsx

```diff
@@ -11,22 +11,25 @@ export function Default() {
 
   const handleToggle = useCallback(() => setActive((active) => !active), []);
 
-  const contentStatus = active ? 'Deactivate' : 'Activate';
-  const textStatus = active ? 'activated' : 'deactivated';
+  const contentStatus = active ? 'Turn off' : 'Turn on';
 
   return (
     <SettingToggle
+      title="Multipass"
+      description="Allow customers to log in with an external customer account system."
+      settingStatus={{
+        enabled: {content: 'On'},
+        disabled: {content: 'Off'},
+      }}
       action={{
         content: contentStatus,
         onAction: handleToggle,
       }}
       enabled={active}
     >
-      This setting is{' '}
-      <Text variant="bodyMd" fontWeight="bold" as="span">
-        {textStatus}
-      </Text>
-      .
+      <Box width="100%">
+        <Banner status="info">An Example Banner Child</Banner>
```

Comment:
Could we replace this with more realistic copy to match the rest of our component examples? @deannatron are you interested in writing this?